ballerina/tests/README.MD

```diff
@@ -0,0 +1,88 @@
+# Running Tests
```

Comment:
Let's follow the structure here
https://github.com/ballerina-platform/module-ballerinax-googleapis.gmail/blob/master/ballerina/tests/README.md